examples/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "rustls-symcrypt-examples"
+version = "0.0.1"
+edition = "2021"
+authors = ["nnmkhang"]

```

Comment:
>nnmkhang [](http://example.com/codeflow?start=12&length=8)

Should these repos be moved into the Microsoft org and have Microsoft listed as the author?

Reply:
Will do for this one and for rust-symcrypt